src/xpk/parser/cluster.py

```diff
@@ -169,25 +170,95 @@ def set_cluster_parser(cluster_parser):
       help='The tpu type to use, v5litepod-16, etc.',
   )
 
+  ### "cluster create-ray" command parser
+
+  cluster_create_ray_cluster_parser = cluster_subcommands.add_parser(
+      'create-ray',
+      help='Create RayCluster',
+  )
+  cluster_create_ray_cluster_required_arguments = (
+      cluster_create_ray_cluster_parser.add_argument_group(
+          'Required Arguments',
+          'Arguments required for cluster create-ray-cluster.',
+      )
+  )
+  cluster_create_ray_cluster_optional_arguments = (
+      cluster_create_ray_cluster_parser.add_argument_group(
+          'Optional Arguments',
+          'Arguments optional for cluster create-ray-cluster.',
+      )
+  )
+  cluster_create_ray_cluster_capacity_arguments = (
+      cluster_create_ray_cluster_parser.add_argument_group(
+          'Capacity Arguments',
+          'Arguments related to capacity for cluster create-ray-cluster.',
+      )
+  )
+  cluster_create_ray_cluster_tensorboard_arguments = (
+      cluster_create_ray_cluster_parser.add_argument_group(
+          'Optional Vertex AI Tensorboard Arguments',
+          'Arguments for creating Vertex AI Tensorboard in cluster create.',
+      )
+  )
+
+  ### RayCluster required arguments specific to "cluster create"
+  cluster_create_ray_cluster_required_arguments.add_argument(
+      '--tpu-type',
```

Comment:
might want to add --device-type in the future to support GPUs/CPUs. def not needed now

Reply:
Good call. Added a TODO